include/oneapi/tbb/parallel_reduce.h

```diff
@@ -35,9 +35,9 @@ inline namespace d0 {
 
 template <typename Body, typename Range>
 concept parallel_reduce_body = splittable<Body> &&
-                               requires( Body& body, const Range& range, Body& rhs ) {
+                               requires( Body& body, const Range& range, Body&& rhs ) {
                                    body(range);
-                                   body.join(rhs);
+                                   body.join(std::move(rhs));
```

Comment:
Since this part of the reduce API was not changed at all, it was a mistake to change this concept. This line was removed.
`parallel_reduce_reduction` and `parallel_reduce_combine` concepts were changed instead.